### ISSUE TITLE
Can't figure this out! TOKENIZATION FAILED: Unable to create tensor, you should probably activate truncation and/or padding...

### ISSUE THREAD

**Issue by BBC-Esq** [UKPLab/sentence-transformers#3397]
Here is the tail end.

TOKENIZATION FAILED: Unable to create tensor, you should probably activate truncation and/or padding with 'padding=True' 'truncation=True' to have batched tensors with the same length. Perhaps your features (`attention_mask` in this case) have excessive nesting (inputs type `list` where type `int` is expected).

I've tried for days to fix this...

1) passed padding and truncation parameters within "tokenizer_kwargs", but it seems that ST already sets them?
2) Tried using fast/slow tokenizers due to reports claiming that the rust backend for tokenizers has problems...
3) Tried to change my multiprocessing and threading around because of seg faults...Keep getting errors like the one below periodically, and then it's back to the padding/truncation issue...
4) Tried different library versions...
5) I'd be happy to share code and would owe someone bigtime if they can help.  I've already wasted multiple days to no avail...

Here's just one example of the seg fault I keep getting:  I obtained by using this in the entry script to my program:

```
import faulthandler, signal
faulthandler.enable(all_threads=True)
```

```
Windows fatal exception: access violation

Thread 0x0000695c (most recent call first):
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 579 in _handle_results
  File "C:\Program Files\Python312\Lib\threading.py", line 1012 in run
  File "C:\Program Files\Python312\Lib\threading.py", line 1075 in _bootstrap_inner
  File "C:\Program Files\Python312\Lib\threading.py", line 1032 in _bootstrap

Thread 0x00009c70 (most recent call first):
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 531 in _handle_tasks
  File "C:\Program Files\Python312\Lib\threading.py", line 1012 in run
  File "C:\Program Files\Python312\Lib\threading.py", line 1075 in _bootstrap_inner
  File "C:\Program Files\Python312\Lib\threading.py", line 1032 in _bootstrap

Thread 0x00008ec8 (most recent call first):
  File "C:\Program Files\Python312\Lib\multiprocessing\connection.py", line 1016 in _exhaustive_wait
  File "C:\Program Files\Python312\Lib\multiprocessing\connection.py", line 1084 in wait
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 502 in _wait_for_updates
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 522 in _handle_workers
  File "C:\Program Files\Python312\Lib\threading.py", line 1012 in run
  File "C:\Program Files\Python312\Lib\threading.py", line 1075 in _bootstrap_inner
  File "C:\Program Files\Python312\Lib\threading.py", line 1032 in _bootstrap

Thread 0x0000be8c (most recent call first):
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 114 in worker
  File "C:\Program Files\Python312\Lib\threading.py", line 1012 in run
  File "C:\Program Files\Python312\Lib\threading.py", line 1075 in _bootstrap_inner
  File "C:\Program Files\Python312\Lib\threading.py", line 1032 in _bootstrap

Thread 0x0000b0b4 (most recent call first):
  File "C:\Program Files\Python312\Lib\multiprocessing\pool.py", line 114 in worker
  File "C:\Program Files\Python312\Lib\threading.py", line 1012 in run
  File "C:\Program Files\Python312\Lib\threading.py", line 1075 in _bootstrap_inner
  File "C:\Program Files\Python312\Lib\threading.py", line 1032 in _bootstrap

Current thread 0x00005c30 (most recent call first):
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\sentence_transformers\SentenceTransformer.py", line 1576 in _text_length
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\sentence_transformers\SentenceTransformer.py", line 652 in encode
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\langchain_huggingface\embeddings\huggingface.py", line 127 in _embed
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\langchain_huggingface\embeddings\huggingface.py", line 150 in embed_documents
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\database_interactions.py", line 497 in create_database
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\torch\utils\_contextlib.py", line 116 in decorate_context
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\database_interactions.py", line 703 in run
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\Lib\site-packages\torch\utils\_contextlib.py", line 116 in decorate_context
  File "D:\Scripts\VectorDB-Plugin-for-LM-Studio\working\database_interactions.py", line 315 in create_vector_db_in_process
  File "C:\Program Files\Python312\Lib\multiprocessing\process.py", line 108 in run
  File "C:\Program Files\Python312\Lib\multiprocessing\process.py", line 314 in _bootstrap
  File "C:\Program Files\Python312\Lib\multiprocessing\spawn.py", line 135 in _main
  File "C:\Program Files\Python312\Lib\multiprocessing\spawn.py", line 122 in spawn_main
  File "<string>", line 1 in <module>
```
